src/fake_keyboard.cpp

```diff
@@ -145,14 +146,16 @@ bool fake_keyboard::is_legacy_supported() {
 // Copied from https://github.com/ReimuNotMoe/ydotool
 void fake_keyboard::ydo_uinput_emit(uint16_t type, uint16_t code, int32_t val,
                                     bool syn_report) const {
-    input_event ie{};
+    struct input_event ie;
+    std::memset(&ie, 0, sizeof(ie)); // zero timeval and other fields
```

Comment:
```suggestion
    input_event ie{};
```

Default initialization in c++ has exactly the same effect as zeroing with memset. I would prefer to not change it.

Reply:
Understood! 

I was debugging why the code wasn't working on my machine. I thought it could be something related to memory, then I saw it was not, and I forgot to remove this change.